### ISSUE TITLE
Use a module loader

### ISSUE THREAD

**Issue by sigurdsvela** [Riiskits/Riiskit-Child#2]
Use requireJS as module loader, and do not concatenate the files into a single file

**Comment by sigurdsvela** [comment]
This is how we'll solve the JS portion of this #1 issue

**Comment by Jeger** [comment]
How does this affect performance? Worth considering that different types of websites / apps work differently. 
http://code.hootsuite.com/why-using-requirejs-could-hurt-your-web-performance/

**Comment by sigurdsvela** [comment]
With one test case showing that this may reduce the number of HTTP requests with 98%, [requirejs optimiser might just be the answer to you pessimism](https://blog.serverdensity.com/improving-frontend-web-performance-using-requirejs-optimiser/).

**Comment by sigurdsvela** [comment]
As well; A module loader is needed(So we can use the import and export functionality of ES6), and all module loaders have this issue, the only possible solution seems to be to use some optimiser.

**Comment by Jeger** [comment]
Agreed, and as far as i can tell from limited reading on the subject this is only really necessary if the project relies on ES for core functionality.
